.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Let's add the `.idea` directory to the `.gitignore` file and remove it from the repo.